charts/lagoon-core/templates/api.deployment.yaml

```diff
@@ -70,6 +70,8 @@ spec:
         - name: HARBOR_ADMIN_PASSWORD
           value: {{ . | quote }}
         {{- end }}
+        - name: HARBOR_API_VERSION
+          value: {{ .Values.harborAPIVersion | default "v2.0" | quote }}
```

Comment:
This works, but what we have done for other default values is to set them in `values.yaml` for the chart instead of putting the value itself in the template. Could we do it that way instead?

For example see the `rabbitMQUsername`, `elasticsearchScheme`, `elasticsearchHostPort` values.

Reply:
+1 to this